nf_core/__main__.py

```diff
@@ -1622,6 +2014,56 @@ def bump_version(new_version, dir, nextflow):
         sys.exit(1)
 
 
+# nf-core create-logo
+
+
+@nf_core_cli.command("create-logo")
+@click.argument("logo-text", metavar="<logo text>")
+@click.option("-d", "--dir", type=click.Path(), default=".", metavar="<directory>")
```

Comment:
Maybe filename instead of directory, as we're only creating one file?

Reply:
Nope, that's the location where the file will be saved

Reply:
Is there any way to specify a filename for the output then? Seems strange to not have an option to do that?

Reply:
added